pandas/tests/series/test_constructors.py

```diff
@@ -1390,10 +1390,14 @@ def test_constructor_cast_object(self, index):
         tm.assert_series_equal(s, exp)
 
     @pytest.mark.parametrize("dtype", [np.datetime64, np.timedelta64])
-    def test_constructor_generic_timestamp_no_frequency(self, dtype):
+    def test_constructor_generic_timestamp_no_frequency(self, dtype, request):
```

Comment:
Is this just an alternative to an imperative call that would fail if the test doesn't produce an expected failure?

Reply:
correct

Reply:
nice. more verbose but probably a better general solution